features/detect_anr.feature

```diff
@@ -11,6 +11,20 @@ Scenario: Sleeping the main thread with pending touch events when autoDetectAnrs
     And the request is valid for the error reporting API version "4.0" for the "Android Bugsnag Notifier" notifier
     And the exception "errorClass" equals "ANR"
     And the exception "message" starts with " Input dispatching timed out"
+    And the exception "type" equals "android"
+
+@skip_android_8_1
+Scenario: Sleeping the main thread with pending touch events when autoDetectAnrs = true
```

Comment:
It would be good to rename this scenario and the previous one to indicate that one triggers an ANR in the NDK layer, and one triggers an ANR in the JVM layer.